starter-code/public/javascript/index.js

```diff
@@ -2,22 +2,22 @@ const charactersAPI = new APIHandler("http://localhost:8000")
 
 $(document).ready( () => {
   document.getElementById('fetch-all').onclick = function(){
-
```

Comment:
Instead of just calling the functions on click you can also gather all the needed information here from the DOM and then pass that information as parameters into the function.